packages/terra-data-grid/src/FlowsheetDataGrid.jsx

```diff
@@ -62,6 +62,11 @@ const propTypes = {
    */
   rowHeight: PropTypes.string,
 
+  /**
+   * String that specifies the minimum height for the rows on the table. rowHeight takes precedence if valid CSS value is passed.
```

Comment:
It would probably be helpful to add that this setting allows the cell to grow to fit the cell content.

Reply:
Done.
commit: 3173b8e8bc4563c520944f3ff6c29b90f3c9124c
Thanks